modular_skyrat/modules/morenarcotics/code/reagents/meth.dm

```diff
@@ -0,0 +1,169 @@
+/datum/chemical_reaction/crystal_meth
+	required_reagents = list(/datum/reagent/drug/methamphetamine = 10, /datum/reagent/hydrogen = 5, /datum/reagent/chlorine = 5)
```

Comment:
Make these lists multi-line, I don't want to repeat this comment on each of them.